src/main/java/slimeknights/tconstruct/TConstruct.java

```diff
@@ -93,6 +101,7 @@ public static void init(final InterModEnqueueEvent event) {
   @SubscribeEvent
   public static void postInit(final InterModProcessEvent event) {
     proxy.postInit();
+    proxy.debug();
```

Comment:
It would be worth considering ditching the proxy model. In a lot of my other 1.14 mods I went the route of a client event handler which subscribes to events such as InterModProcessEvent.

Reply:
Sounds good, separate PR